### DISCUSSION TITLE
Changed File Location

### DISCUSSION THREAD

**Discussion by DeathTank342** [aclap-dev/video-downloadhelper#166] [Help and Q&A]
### Your question

I have been using VDH for awhile to save movies for personal use, but lately the file location has changed from downloads to an entirely different folder in my C drive when it used to be in a totally different hard drive. Is there anyway to manually change file location? 

### Addon & CoApp version and details

Google Chrome, Video DownloadHelper Version 8.2.0.24

**Comment by GrampaWildWilly** [comment]
If you use the CoApp & Download (not Quick Download), you can tell VDH the directory to download into.  If you want to always get the file selection dialog, make Download your default action.

**Comment by paulrouget** [comment]
Hey, this will be fixed in 8.2.1. Coming soon.

@GrampaWildWilly that's for in-browser download, not via coapp.
